src/ps_wait.js

```diff
@@ -0,0 +1,64 @@
+'use strict'
+
+const cli = require('heroku-cli-util')
+
+module.exports = function * (context, heroku, delay) {
```

Comment:
I like the idea of adding `--wait` to restart, but right now this helper kinda looks like a utility function and kinda like a command. Should we extract flag parsing (and maybe `exit` handling--Just™ throw instead?) to make this a more generic?

Reply:
we could reuse this code too https://github.com/heroku/heroku-apps/blob/2aa083ee9b852cb2c5202a69fb63b05b4db60036/src/releases.js